notes/stories/television/tv-theamericanshortstory.st.txt

```diff
@@ -270,3 +270,51 @@ childishness [Andy was rude to the cigar lady and the bartender, and his bellico
+superior and subordinate [The story follows Mrs. Mcintyre was bossing around the workers on her farm throughout the story.]
+religious hypocrisy [Mrs. McIntyre spoke often of her Christian values in hiring black people and refugees, but crapped on them and criticized them when it suited her. In an ultimate act of religious hypocrisy, she failed to warn Mr. Guizac that an out of control tractor was rolling his way and he was crushed to death under its wheels as a result.]
+hypocritical character [Mrs. McIntyre spoke often of her Christian values in hiring black people and refugees, but crapped on them and criticized them when it suited her.]
+what it is like to be a refugee [The titular "displaced person" Mr. Guizac relocated with his family to a farm in Georgia in the aftermath of the Holocaust.]
+
+:: Minor Themes
+facing job insecurity [The Shortleys felt threatened by the newly arrived and comparatively hard working farmhand Mr. Guizac and tried their best to get him fired.]
+religious occupation [The Catholic priest Father Flynn was going about doing priestly things.]
+The Holocaust [Mr. Guizac was from that place where "where all them bodies was stacked up at" said Mrs. Shortley in reference to this genocide.]
+husband and wife [The Shortleys bugged out and fled the farm with their two daughters.]
+mother and daughter [Mrs. Shortley bolted from the farm with her two young daughters.]
+father and daughter [Mr. Shortley bolted from the farm with his two young daughters.]
+Catholicism [Father Flynn gave the Last Rights to Mr. Guizac as he lay dying under a tractor. He also spoke about purgatory at several points in the story.]
+murder [A bitter and resentful Mr. Shortley positioned a tractor to roll over Mr. Guizac as he worked beneath another machine as if by accident. Mrs. McIntyre stood idly by as this transpired, making her complicit in the killing.]
+class prejudice in society [The farm owner Mrs. McIntyre condescended one of her farmhands by telling him that "what you common people don't realize is that I'm the one who holds all the stings together" and went on to smugly explain that they were all dependent on her.]
+foreign point of view [The Holocaust survivor Mr. Guizac was puzzled to discover that interracial marriage was de facto forbidden in the region of the United States where he'd emigrated to.]
+xenophobia in society [Mrs. Shortley was paranoid about Polish immigrants taking over her livelihood and overrunning her community. Mr. Shortley made some bigoted remarks about foreigners.]
+religious fervor [Mrs. Shortley wandered out to an open field and began spouting some evidently deranged religious prophesy to herself.]
+World War II [Mr. Guizac was a Holocaust survivor. Some passing remarks were make about the fighting that'd transpired in this war.]
+discrimination in society [The black farmhand Sulk was pointedly denied the right to marry a Polish woman.]
```

Comment:
Is this not a kind of "racial segregation" thing more than discrimination? Or something about racial purity (racial supremacism perhaps)? It just sounds a little peculiar to say that it's "discriminatory" for one kind of person to be able to marry Polish women while another kind of person may not marry Polish women because "Polish women", I hope, themselves have a say in the matter. "Discrimination" is usually about who's preferred for a job and such no?

Reply:
Wikipedia has a lengthy article on [laws prohibiting interracial marriage in the United States](https://en.wikipedia.org/wiki/Anti-miscegenation_laws_in_the_United_States) that is part of a series on discrimination. so I think as long as we have the theme **discrimination in society** it should apply here.

But I do reckon the **racial segregation** and **racial supremacy** themes could be motivated.